.travis.yml

```diff
@@ -21,7 +21,7 @@ _bindings:
     if: env(BUILD_ONLY) IS NOT present || env(BUILD_ONLY) =~ /python/
   ruby: &ruby
     language: ruby
-    rvm: 2.0.0-p648
+    rvm: 2.3.8
```

Comment:
If in selenium4 we're going to up this. It feels a good time to introduce at least testing for higher versions such as 2.4 / 2.5 (I know historically selenium has never supported the top edge?

Thoughts @p0deje ?

Reply:
I'm not sure if Thomas and Titus agreed to release it before Selenium 4, but if they did, versions will be upgraded everywhere.

Reply:
@luke-hill the only issue with testing multiple versions is the way the test matrix is built - and we should be testing 2.6 too -- there is zero reason not to officially support it